### DISCUSSION TITLE
草稿在生产环境不能正常生效

### DISCUSSION THREAD

**Discussion by ericting** [hugo-fixit/FixIt#113] [Q&A] [answered]
### Describe the bug 描述你遇到的错误

- 在``hugo new``一篇文章后，启用``draft: true``在本地正常，但是由verlcel部署的hugo--``version 0.91.0``静态网页，参数并不生效。

**Comment by ericting** [comment]
也就是说本地能够正常开启草稿，但是由vercel部署的网站——生产环境，草稿貌似不生效

**Comment by Lruihao** [comment, marked as answer]
hugo 的设计就是草稿不在生产环境显示，我认为这种设计是合理，草稿才有草稿的意义。

另外，追随这种思想，FixIt v0.2.14 的菜单也可以设置为 草稿，同样在生产环境不会生效

**Comment by ericting** [comment]
![image](https://user-images.githubusercontent.com/29013507/163133711-9b582a6b-bd0f-4f18-8042-ddfabea867d0.png)
可能我表达的有点问题~你看看这个，就是我设置成了草稿。但在生产环境下依旧显示该文章，尝试清除了缓存，也还是一样显示出来了

**Reply by Lruihao** [reply]
你的自动构建命令是什么

**Reply by Lruihao** [reply]
构建命令不要带 `--buildDrafts` 参数

**Reply by ericting** [reply]
> 你的自动构建命令是什么

![image](https://user-images.githubusercontent.com/29013507/163135498-c04ea0d2-e9f3-4cdc-a9c4-a91f244d161a.png)
这个，默认的

**Reply by ericting** [reply]
好的谢谢

**Reply by ericting** [reply]
> 构建命令不要带 `--buildDrafts` 参数

查了下-D 就是默认 -D, ``--buildDrafts            include content marked as draft``

**Reply by Lruihao** [reply]
-D 就是 `--buildDrafts` 的缩写表示构建草稿，你手动改一下参数 

```bash
hugo --gc
```
